packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textEditorParts/mask.tests.js

```diff
@@ -2015,40 +2017,15 @@ QUnit.module('paste', moduleConfig, () => {
 });
 
 QUnit.module('drag text', moduleConfig, () => {
-    QUnit.test('mask should support drag', function(assert) {
-        const $textEditor = $('#texteditor').dxTextEditor({
-            mask: '(XX)',
-            maskRules: {
-                'X': 'x'
-            },
-            value: ' x'
-        });
-
-        const $input = $textEditor.find(`.${TEXTEDITOR_INPUT_CLASS}`);
-
-        $input.val('(x)_').trigger('drop');
-
-        this.clock.tick(10);
-
-        assert.equal($input.val(), '(x_)', 'mask is correct');
-    });
-
-    QUnit.test('mask should support drag with spaces', function(assert) {
-        const $textEditor = $('#texteditor').dxTextEditor({
-            mask: '(XXXX)',
-            maskRules: {
-                'X': /[xy]/
-            },
-            value: ' x y'
-        });
-
+    QUnit.test('drag events should be prevented in a masked text box', function(assert) {
```

Comment:
```suggestion
    QUnit.test('drop event should be prevented in a masked text box (T1188135)', function(assert) {
```

Reply:
Fixed in [f0b0bf5f2a89e12ddc6f0d76ad534fe221a08aef].